test/steps/image_registry.go

```diff
@@ -24,26 +24,46 @@ import (
 	"github.com/cucumber/godog"
 	"github.com/kiegroup/kogito-cloud-operator/test/config"
 	"github.com/kiegroup/kogito-cloud-operator/test/framework"
+	"github.com/kiegroup/kogito-cloud-operator/test/steps/mappers"
 )
 
+/*
+	DataTable for Maven:
+	| -Dproperty=value |
+*/
+
 // registerImageRegistrySteps register all existing image registry steps
 func registerImageRegistrySteps(ctx *godog.ScenarioContext, data *Data) {
 	ctx.Step(`^Local example service "([^"]*)" is built by Maven using profile "([^"]*)" and deployed to runtime registry$`, data.localServiceBuiltByMavenWithProfileAndDeployedToRuntimeRegistry)
+	ctx.Step(`^Local example service "([^"]*)" is built by Maven using profile "([^"]*)" and options and deployed to runtime registry:$`, data.localServiceBuiltByMavenWithProfileAndOptionsAndDeployedToRuntimeRegistry)
```

Comment:
```suggestion
	ctx.Step(`^Local example service "([^"]*)" is built by Maven using profile "([^"]*)" and deployed to runtime registry with maven options:$`, data.localServiceBuiltByMavenWithProfileAndOptionsAndDeployedToRuntimeRegistry)
```
I think we should let the table information be the last part of the step, so we know easily what it is for

Reply:
+1

Reply:
I originally didn't want it that way as it suggests that it is deployed to registry with these options whereas it is just built with these options, but probably still better than the current naming :)